bento-sample-app/src/main/java/com/yelp/android/bentosampleapp/ListViewActivity.kt

```diff
@@ -90,6 +91,26 @@ class ListViewActivity : AppCompatActivity() {
         }
     }
 
+    private fun addListComponentRemovableItems(controller: ComponentController) {
+        val component = ListComponent(null, ListComponentExampleViewHolder::class.java)
+        with(component) {
+            setStartGap(50)
+            setData(listOf("List element 1"))
+            toggleDivider(true)
+            controller.addComponent(this)
+
+            listOf(1, 1, 1, 2, 4).forEachIndexed { index, i -> insertData(i, "List element ${index + 2}") }
+            insertData(0, "List element First")
+            insertData(0, "List element First")
+            insertData(1 + component.count / 2, "List element Last")
+            removeData("List element First")
+            val index = removeData("List element 1")
+            insertData(index, "List element 1")
+            removeData("List element 2")
+            removeData("List element nonexistent")
```

Comment:
I don't think this is a really useful code: you are adding and removing items even before the component would be displayed on screen. You might want to consider instead first displaying a list of items, and adding a button somewhere to try the scenario where you remove items dynamically.